plugins/techdocs-addons/src/components/TechDocsReaderPage/TechDocsReaderPage.tsx

```diff
@@ -33,6 +34,8 @@ import { TechDocsReaderPageSubheader } from '../TechDocsReaderPageSubheader';
  * @public
  */
 export type TechDocsReaderPageProps = {
+  hideHeader?: boolean;
+  addonConfig?: React.ReactNode;
```

Comment:
This looks very similar to the `TechDocsAddonConfig` type? Maybe reuse? both terminology (to avoid confusion with app config) and type.